README.md

```diff
@@ -5,7 +5,8 @@
 [![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fstryker-mutator%2Fstryker-net%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker-net/master)
 [![Slack](https://img.shields.io/badge/chat-on%20slack-blueviolet?style=flat-square)](https://join.slack.com/t/stryker-mutator/shared_invite/enQtOTUyMTYyNTg1NDQ0LTU4ODNmZDlmN2I3MmEyMTVhYjZlYmJkOThlNTY3NTM1M2QxYmM5YTM3ODQxYmJjY2YyYzllM2RkMmM1NjNjZjM)
 
-# ![S](https://raw.githubusercontent.com/stryker-mutator/stryker-mutator.github.io/master/images/stryker-80x80.png)tryker.NET
+# ![S](https://raw.githubusercontent.com/stryker-mutator/stryker-mutator.github.io/develop/static/images/stryker-80x80.png)tryker.NET
```

Comment:
```suggestion
# ![S](https://raw.githubusercontent.com/stryker-mutator/stryker-mutator.github.io/master/static/images/stryker-80x80.png)tryker.NET
```

Reply:
there's no `master` branch on the `github.io` repository? https://github.com/stryker-mutator/stryker-mutator.github.io/branches/all

Reply:
Ah I see, I had not realized we had stopped using branch-based github pages.